org.ekstep.sunbirdcommonheader-1.9/editor/tocErrorPopupTemplate.html

```diff
@@ -13,7 +13,7 @@ <h5 class="ui header errMessage">
         </div>
         <div class="actions">
             <div class="ui buttons ">
-                <button id="closePopUp " class="ui blue button" type="button " ng-click="closeThisDialog();">Close</button>
+                <button id="closePopUp " class="ui blue button" type="button " ng-click="closePopup();">Close</button>
```

Comment:
When user clicks on close button also, popup will not close because
$scope.closePopup = function() {
                            $scope.closeThisDialog()
                            mainCtrl.updateToc();
                        }

mainCtrl.updateToc(); will open the popup again. This is wrong.